StackoverflowChatbot/CommandProcessors/PriorityProcessor.cs

```diff
@@ -113,131 +75,21 @@ private bool TryGetNativeCommand(EventData data, out IAction? action)
 				return action != null;
 			}
 
+			// Why is action getting assigned but is unused?
 			action = null;
 			return false;
 		}
 
-		private BaseCommand CreateCommandInstance(Type commandType)
-		{
-			var parameterTypes = commandType
```

Comment:
the new comment "todo service locator", which is causing the conflict, is inside an already gone method.